src/components/Profile/ProfileInfo/ProfileInfo.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+import BACKGROUND_IMG from '../../../url_utils/index'
```

Comment:
You don't need to specify "index" file when importing - it is a default import file when importing a folder.

Reply:
/fxd